sdk/identity/Azure.Identity/README.md

```diff
@@ -128,80 +129,113 @@ var credential = new ChainedTokenCredential(new ManagedIdentityCredential(), new
 
-## Credential Classes
+### Examples
 
-### Authenticating Azure Hosted Applications
+These examples demonstrate authenticating the `SecretClient` from the [Azure.Security.KeyVault.Secrets][secrets_client_library] client library using the `ManagedIdentityCredential`.
 
-|Credential  | Usage
-|-|-
-|[`DefaultAzureCredential`][ref_DefaultAzureCredential]|provides a simplified authentication experience to quickly start developing applications run in the Azure cloud
-|[`ChainedTokenCredential`][ref_ChainedTokenCredential]|allows users to define custom authentication flows composing multiple credentials
-|[`ManagedIdentityCredential`][ref_ManagedIdentityCredential]|authenticates the managed identity of an azure resource
-|[`EnvironmentCredential`][ref_EnvironmentCredential]|authenticates a service principal or user via credential information specified in environment variables
+#### Authenticate with a user-assigned managed identity
+
+```c#
+var credential = new ManagedIdentityCredential(clientId: "<USER ASSIGNED MANAGED IDENTITY CLIENTID>");
+var client = new SecretClient(new Uri("https://myvault.vault.azure.net/"), credential);
+```
 
-### Authenticating Service Principals
+#### Authenticate with a system-assigned managed identity
 
-|Credential  | Usage
-|-|-
-|[`ClientAssertionCredential`][ref_ClientAssertionCredential]|authenticates a service principal using a signed client assertion
-|[`ClientCertificateCredential`][ref_ClientCertificateCredential]|authenticates a service principal using a certificate
-|[`ClientSecretCredential`][ref_ClientSecretCredential]|authenticates a service principal using a secret
+```c#
+var credential = new ManagedIdentityCredential()
+var client = new SecretClient(new Uri("https://myvault.vault.azure.net/"), credential);
+```
 
-### Authenticating Users
+## Cloud configuration
 
-|Credential  | Usage
-|-|-
-|[`AuthorizationCodeCredential`][ref_AuthorizationCodeCredential]|authenticates a user with a previously obtained authorization code
-|[`DeviceCodeCredential`][ref_DeviceCodeCredential]|interactively authenticates a user on devices with limited UI
-|[`InteractiveBrowserCredential`][ref_InteractiveBrowserCredential]|interactively authenticates a user with the default system browser
-|[`OnBehalfOfCredential`][ref_OnBehalfOfCredential]|propagates the delegated user identity and permissions through the request chain
-|[`UsernamePasswordCredential`][ref_UsernamePasswordCredential]|authenticates a user with a username and password
+Credentials default to authenticating to the Azure AD endpoint for
+Azure Public Cloud. To access resources in other clouds, such as Azure Government
+or a private cloud, configure credentials with the `authority` argument.
+[AzureAuthorityHosts](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.azureauthorityhosts?view=azure-dotnet)
+defines authorities for well-known clouds:
+```c#
```

Comment:
We'll need to make these snippets so that the code is compiled, and smoke tested to avoid getting stale.   Please see: [Updating Sample Snippets](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets).